tsuru/client/apps.go

```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
 	return id
 }
 
-func (a *app) String() string {
-	format := `{{ if .Error -}}
+const simplifiedFormat = `{{ if .Error -}}
 Error: {{ .Error }}
 {{ end -}}
 Application: {{.Name}}
-Description:{{if .Description}} {{.Description}}{{end}}
-Tags:{{if .TagList}} {{.TagList}}{{end}}
+{{- if .Description }}
+Description: {{.Description}}
+{{- end }}
+{{- if .TagList }}
+Tags: {{.TagList}}
+Created by: {{.Owner}}
+{{- end }}
+Platform: {{.Platform}}
+Plan: {{ .Plan.Name }}
+Pool: {{.Pool}} ({{ .Provisioner }}{{ if .Cluster}} cluster: {{ .Cluster }}{{end}})
+{{if not .Routers -}}
```

Comment:
If .Routers exist, shouldn't if be listed also?

Reply:
.Router is a legacy method from old tsuru apis, @tcarreira